httpd/handler_test.go

```diff
@@ -624,7 +624,7 @@ func TestHandler_CreateUser_BadRequest(t *testing.T) {
 }
 
 func TestHandler_CreateUser_BadRequest_NoName(t *testing.T) {
-	srvr := OpenServer(NewMessagingClient())
+	srvr := OpenAuthlessServer(NewMessagingClient())
```

Comment:
Minor: I do think "Unitialized" was helpful. Perhaps call the function `OpenAuthlessUninitServer()`? Not sure.